cpp/include/raft/neighbors/detail/cagra/search_plan.cuh

```diff
@@ -250,17 +250,10 @@ struct search_plan_impl : public search_plan_impl_base {
     }
   }
 
-  void check(uint32_t topk)
+  virtual void check(const uint32_t topk)
   {
+    // For multi-CTA and multi kernel
```

Comment:
Isn't it for single-CTA and multi-kernel since it is being overriden in multi-CTA?